ronin-chain/katana-operation-contracts
# feat: fork Uniswap/universal-router

### Description
Clone [universal-router](https://github.com/Uniswap/universal-router) to the `src/aggregate-router` directory.

#### How to verify?
1. Get the checksum of [universal-router](https://github.com/Uniswap/universal-router) at tag `v1.6.0`
```
git clone https://github.com/Uniswap/universal-router&& cd universal-router/contracts && git checkout v1.6.0
find . -type d -name 'test' -prune -o -type f -print | sort | xargs shasum -a 256 | shasum -a 256

```
It should output `9b2baf032688cd28e194c7982d480fea719ade5005b33b1ae1e5d162d0b92ef0`.

2. Verify
Make sure that your working directory is the root of this repository.
```
cd src/aggregate-router && find . -type f | sort | xargs shasum -a 256 | shasum -a 256
```
